src/test/groovy/com/gmaslowski/spock/sample/GroovyDummyServiceTest.groovy

```diff
@@ -45,4 +46,16 @@ class GroovyDummyServiceTest extends Specification {
         then:
         2 * dummyRepository.getDummyById(dummyId)
     }
+
+    def "should not deactivate dummy with negative id"() {

```

Comment:
The aim of the project was to show differences between java and groovy + spock testing. Could you a java example?